js/lib/beautify-html.js

```diff
@@ -84,7 +84,7 @@
         return s.replace(/\s+$/g, '');
     }
 
-    function style_html(html_source, options, js_beautify, css_beautify) {
+    function style_html(html_source, options, js_beautify_raw, css_beautify_raw) {
```

Comment:
But they can all use the same configuration file or options object.  So the same schema needs to work for all of them.